reviews three/swagger.yaml

```diff
@@ -0,0 +1,954 @@
+          
+        
+          | `Note`| The array of line items on a draft order must not contain a `variant_id` more than once. The data copied over from a variant upon creation of a line item is never subsequently updated.  |
+          |---|-------------|
+          
+          #### About Custom Shipping Lines
+          
+            Using the Draft Orders API provides you with the flexibility to fulfill order shipments either using Shopify or outside of Shopify.  To fulfill your order within Shopify, you can send the `handle` of the `shipping_rate` set via the Shopify admin.  For a custom shipping line, simply specify a `shipping_line` object with no `handle`.  This is covered in more detail in the endpoint calls below.  
+         
+          
+      tags:
+        - Create
+
+      parameters:
+        - name: draft_order
+          in: body
+          description: draft_order to create.
+          schema:
+            allOf:
+              - $ref: '#/definitions/DraftOrder'
+            required:
+            - line_items
+      
+          
+            
+
+      responses:
+        '200':
+          description: A single DraftOrder
+          schema:
+            $ref: '#/definitions/DraftOrder'
+            
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X POST -H "Content-Type: application/json" -H "Authorization: Basic YOUR-API-KEY=" -d '{
+              "id": 57760135,
+              "order_id": 57760134,
+              "name": "api created draft order",
+              "customer": {
+                "accepts_marketing": true,
+                "created_at": "2016-11-18T17:10:07Z",
+                "email": "string",
+                "first_name": "string",
+                "order_id": 0,
+                "last_name": "string",
+                "note": "string",
+                "orders_count": 0,
+                "state": "string",
+                "total_spent": 0,
+                "updated_at": "2016-11-18T17:10:07Z",
+                "tags": "string"
+              },
+              "shipping_address": "string",
+              "billing_address": "string",
+              "note": "string",
+              "email": "string",
+              "currency": "string",
+              "invoice_sent_at": "2016-11-18T17:10:07Z",
+              "invoice_url": "string",
+              "line_items": [
+                {
+                  "custom": true,
+                  "id": "string",
+                  "vendor": "string",
+                  "quantity": "string",
+                  "gift_card": true,
```

Comment:
ok... so, the doc has a column for what's read only. You should review the design doc and remove anything from the create POST which is read_only. 

Really a basic create post has line items, with a variant id and quantity OR a title, price and quantity